tests/chart_tests/test_astronomer_commander.py

```diff
@@ -63,7 +63,7 @@ def test_astronomer_commander_deployment_default(self, kube_version):
         assert env_vars["COMMANDER_REGION"] == "us-west-2"
         assert env_vars["COMMANDER_BASE_DOMAIN"] == "custom-dp-123.example.com"
         assert env_vars["COMMANDER_DATAPLANE_MODE"] == "data"
-        assert env_vars["COMMANDER_HOUSTON_JWKS_ENDPOINT"] == "https://houston.example.com/auth"
+        assert env_vars["COMMANDER_HOUSTON_JWKS_ENDPOINT"] == "https://houston.example.com"
```

Comment:
Is this really the endpoint? Is this not just the URL? Should we change this ENV VAR? CC @pgvishnuram

Reply:
I don't want this to block this PR, but calling a base domain name `ENDPOINT` is inaccurate.